version.py

```diff
@@ -1 +1 @@
-__version__ = "0.6.3"
+__version__ = "0.7.0"
```

Comment:
Technically breaking, right? So major?

Reply:
Don't be afraid of v1 - it does not mean final 😂
https://0ver.org/

Reply:
hehe, ill do that when I get the unit template stuff in ;)